src/apps/frontend/types/account.ts

```diff
@@ -1,19 +1,26 @@
+import { PhoneNumber } from './auth';
 import { JsonObject } from './common-types';
 
 export class Account {
   id: string;
   firstName: string;
   lastName: string;
+  phoneNumber: PhoneNumber;
   username: string;
 
   constructor(json: JsonObject) {
     this.id = json.id as string;
     this.firstName = json.firstName as string;
     this.lastName = json.lastName as string;
+    this.phoneNumber = json.phoneNumber as PhoneNumber;
     this.username = json.username as string;
   }
 
   displayName(): string {
     return (`${this.firstName} ${this.lastName}`).trim();
   }
+
+  displayPhoneNumber(): string {
```

Comment:
Think this function should exist in `PhoneNumber`

Reply:
modified existing PhoneNumber type to get this functionality.